modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/incubus_draft.dm

```diff
@@ -135,26 +136,32 @@
 	return ..()
 
 /datum/reagent/drug/aphrodisiac/incubus_draft/overdose_effects(mob/living/carbon/human/exposed_mob)
+	// Check if overdosing on incubus draft and succubus milk simultaneously, to prevent chat spam
+	var double_dosing = FALSE
+	for(var/r in exposed_mob.reagents.reagent_list)
```

Comment:
- better var names please
- you can cut out this entire section by replacing it with `if(locate(/datum/reagent/whatever_succubus_is) in exposed_mob.reagents.reagent_list)`

Reply:
Fair, that is a better way of doing that.

Reply:
It's two less lines of code, but I see your point. Done and done!